src/preact/base-element.js

```diff
@@ -611,7 +561,7 @@ export class PreactBaseElement extends BaseElement {
     }
   }
 
-  /** @private */
+  /** @protected */
```

Comment:
is this change intentional?

Reply:
Yes it's used It's used in mutatedAttributesCallback [here](https://github.com/ampproject/amphtml/pull/37309/files#diff-7d1d95d912247f7142f4a510777181aff5dfc1975f5ad4de225cd2dc6b064015R19)